routemaster/client.rb

```diff
@@ -30,36 +30,36 @@ def configure
         end
       end
 
-      def created(topic, callback, timestamp = nil)
-        _send_event('create', topic, callback, timestamp)
+      def created(topic, callback, timestamp = nil, t: nil)
+        _send_event('create', topic, callback, t: t || timestamp)
       end
 
-      def created_async(topic, callback, timestamp = nil)
-        _send_event('create', topic, callback, timestamp, async: true)
+      def created_async(topic, callback, timestamp = nil, t: nil)
+        _send_event('create', topic, callback, t: t || timestamp, async: true)
       end
 
-      def updated(topic, callback, timestamp = nil)
-        _send_event('update', topic, callback, timestamp)
+      def updated(topic, callback, timestamp = nil, t: nil)
+        _send_event('update', topic, callback, t: t || timestamp)
       end
 
-      def updated_async(topic, callback, timestamp = nil)
-        _send_event('update', topic, callback, timestamp, async: true)
+      def updated_async(topic, callback, timestamp = nil, t: nil)
+        _send_event('update', topic, callback, t: t || timestamp, async: true)
       end
 
-      def deleted(topic, callback, timestamp = nil)
-        _send_event('delete', topic, callback, timestamp)
+      def deleted(topic, callback, timestamp = nil, t: nil)
+        _send_event('delete', topic, callback, t: t || timestamp)
       end
 
-      def deleted_async(topic, callback, timestamp = nil)
-        _send_event('delete', topic, callback, timestamp, async: true)
+      def deleted_async(topic, callback, timestamp = nil, t: nil)
+        _send_event('delete', topic, callback, t: t || timestamp, async: true)
       end
 
-      def noop(topic, callback, timestamp = nil)
-        _send_event('noop', topic, callback, timestamp)
+      def noop(topic, callback, timestamp = nil, t: nil)
+        _send_event('noop', topic, callback, t: t || timestamp)
       end
 
-      def noop_async(topic, callback, timestamp = nil)
-        _send_event('noop', topic, callback, timestamp, async: true)
+      def noop_async(topic, callback, timestamp = nil, t: nil)
+        _send_event('noop', topic, callback, t: t || timestamp, async: true)
```

Comment:
Would be good to ad some tests for these changes as they are part of the public interface.